src/javascript/app/Stores/Modules/Statement/Helpers/format-response.js

```diff
@@ -1,27 +1,35 @@
-import { formatMoney } from '_common/base/currency_base';
-import { localize }    from '_common/localize';
-import { toTitleCase } from '_common/string_util';
-import { toMoment }    from 'Utils/Date';
+import { formatMoney }          from '_common/base/currency_base';
+import { localize }             from '_common/localize';
+import { toTitleCase }          from '_common/string_util';
+import { toMoment }             from 'Utils/Date';
+import { getSymbolDisplayName } from '../../Trading/Helpers/active-symbols';
+import { getMarketInformation } from '../../../../Modules/Reports/Helpers/market-underlying';
 
-export const formatStatementTransaction = (transaction, currency) => {
-    const format_string = 'DD MMM YYYY - HH:mm:ss';
-    const transaction_time   = toMoment(transaction.transaction_time).format(format_string);
-    const payout     = parseFloat(transaction.payout);
-    const amount     = parseFloat(transaction.amount);
-    const balance    = parseFloat(transaction.balance_after);
+export const formatStatementTransaction = (transaction, currency, active_symbols = []) => {
+    const format_string           = 'DD MMM YYYY - HH:mm:ss';
+    const transaction_time        = toMoment(transaction.transaction_time).format(format_string);
+    const payout                  = parseFloat(transaction.payout);
+    const amount                  = parseFloat(transaction.amount);
+    const balance                 = parseFloat(transaction.balance_after);
     const should_exclude_currency = true;
+    const shortcode               = ['buy', 'sell'].includes(transaction.action_type) ? transaction.shortcode : null;
+    const display_name            = shortcode ? getSymbolDisplayName(
+        active_symbols,
+        getMarketInformation(shortcode).underlying
+    ) : '';
 
     return {
         action     : localize(toTitleCase(transaction.action_type) /* localize-ignore */), // handled in static_strings_app.js: 'Buy', 'Sell', 'Deposit', 'Withdrawal'
```

Comment:
do we need localize here ?

Reply:
AFAIK this is being localized in the backend.